### DISCUSSION TITLE
How to use tailwindcss color in custom page/custom entry ?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#14440] [Help] [answered]
### Package

Infolist builder

### Package Version

v3.2

### How can we help you?

I am currently try to create timeline using custom entry/custom page for my infolist..

I am generating the custom entry using `php artisan make:infolist-entry Timeline`

my `resource` looks like this:
```
InfolistSection::make('Cronology')
   ->schema([
       Timeline::make('cronology_content')
   ]),
```

in my `timeline.blade.php`

some of other tailwindcss class are available for me.
but for colors its not working

such as:
`<div class="text-green-500">APPROVED</div>`
or
`<div class="text-success-500">APPROVED</div>`

i check with developer console, there's no style with color: green; or similar.

Why is that?
Can someone help?

**Comment by dissto** [comment, marked as answer]
Did you setup a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) ?

**Reply by hazwan91** [reply]
I did...and it still not working...simple class like `text-teal-500` not even working...not even after i ran `npm run build`...

i am still working it out with the custom theme....thanks for pointing me toward something

**Reply by hazwan91** [reply]
oh its working....

need to add this ` "./resources/views/infolists/**/*.blade.php",` to my `tailwind.config.js`

thanks for guiding me right way with simple question
